tests/cli/test_local.py

```diff
@@ -126,40 +132,65 @@ def paramiko_openssh_key_parsing_issue(strict: bool = False):
     )
 
 
+# TODO: This test is incredibly flaky. Needs to be totally reworked.
+@PARAMIKO_SSH_BANNER_BUG
```

Comment:
`@PARAMIKO_SSH_BANNER_BUG` is applied to the test but the code below seams to also specify which tests are expected to fail. Why is that?

Reply:
Yeah I got lazy and just added it to all cases. It's a mess. There's apparently no way to guaranteed that it's not going to happen. Fixed in bc473f890cd570a279cd29ad71e5bb2e5cee0e7f